cloudant-sync-datastore-core/findbugs_excludes.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Copyright (c) 2016 IBM Corp. All rights reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+  ~ except in compliance with the License. You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software distributed under the
+  ~ License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+  ~ either express or implied. See the License for the specific language governing permissions
+  ~ and limitations under the License.
+  -->
+<!--
+A list of known findbugs issues that we cannot fix at this time.
+-->
+<FindBugsFilter>
+
+    <!-- A couple of classes shadow the names of their supertypes.
+    -->
+    <Match>
+        <Bug code="Nm" pattern="NM_SAME_SIMPLE_NAME_AS_SUPERCLASS" />
```

Comment:
Should these be marked as a TODO or tracked in another ticket? I mean we're only excluding them to make findbugs pass, but we should fix the underlying problem eventually.